dandi/files/bases.py

```diff
@@ -469,6 +480,21 @@ def get_validation_errors(
         else:
             # make sure that we have some basic metadata fields we require
             try:
+                # Ensure latest version of NWB Inspector is installed and used client-side
+                current_version, max_version = self._get_inspector_versions()
+                if current_version != max_version:
```

Comment:
Although, perhaps using `<` instead of `!=` here could allow easier testing using dev branches of the NWB Inspector? What do you guys think?